### ISSUE TITLE
500 NLST command not recognized by server

### ISSUE THREAD

**Issue by oz123** [troglobit/uftpd#2]
``` NLST

500 NLST command not recognized by server
```

This is what I get if I try this from python:

```
In [1]: import ftplib
In [2]:  f =ftplib.FTP()
In [46]: f.connect('127.0.0.1', 21 )
Out[46]: '220 uftpd (1.8) ready.'

In [47]: f.retrlines('LIST')
drwxr-xr-x 1     0     0         4096 Jul 11 19:03 .config
drwxr----- 1     0     0           24 Sep 22 21:34 .dbus
drwxr-xr-x 1     0     0           73 May  4 14:11 .dvdrip
drwxr-xr-x 1     0     0         4096 Jun 15 19:49 .fontconfig
...
Out[47]: '226 Transfer complete.'

In [48]: f.nlst()
---------------------------------------------------------------------------
error_perm          
error_perm: 500 NLST command not recognized by server
```

**Comment by troglobit** [comment]
@oz123 Correct, NLST is unfortunately not implemented yet. I'm assuming you need it for a multiple-get operation, right?

**Comment by troglobit** [comment]
The [dev](https://github.com/troglobit/uftpd/tree/dev) branch now has a fix for this issue.  Please let me know if this helps!

**Comment by oz123** [comment]
Hi Joachim, 
Thank you for the quick response. I just build the HEAD of dev branch using latest libite and lastet libuev, and it works really great!
Thank you. 

As for your answer why I need this. I was trying to understand some weird behavior of an old version of ProFTPD server I need to work week. I was looking for an easy alternative to setup and saw your server which came very handy. 

NLST behavs as I would expect it, as the pyftpdlib server I already know. You can see the full details of my problem describe in Stackoverflow here: [List files with pyftp - proftpd vs. pyftpdlib behavior](http://stackoverflow.com/questions/31291617/list-files-with-pyftp-proftpd-vs-pyftpdlib-behavior).
There is a bounty on that question too...

**Comment by troglobit** [comment]
Hi, thanks for testing and reporting this!  I've given a reply at Stackoverflow with the little info I have on the subject :)

I'll be preparing a v1.9 release very soon now to include this, and some other bits and bobs.

Cheers!

**Comment by oz123** [comment]
@troglobit thank you so much!

**Comment by troglobit** [comment]
De nada, @oz123 :smiley:

**Comment by troglobit** [comment]
Reopening ... forgot the fix was still on the `dev` branch.  Will close when having merged to master
for release.

Sorry for the noise!

**Comment by troglobit** [comment]
Merged to master for v1.9 release.
